guide/guide/.js/src/main/scala/io/udash/web/guide/views/frontend/demos/IntroFormDemo.scala

```diff
@@ -0,0 +1,151 @@
+package io.udash.web.guide.views.frontend.demos
+
+import io.udash._
+import io.udash.bootstrap.button.UdashButton
+import io.udash.bootstrap.form.UdashInputGroup
+import io.udash.bootstrap.utils.BootstrapStyles
+import io.udash.bootstrap.utils.BootstrapStyles.Color
+import io.udash.css.CssView
+import io.udash.web.guide.components.BootstrapUtils
+import io.udash.web.guide.demos.AutoDemo
+import io.udash.web.guide.styles.partials.GuideStyles
+import scalatags.JsDom
+
+import scala.collection.mutable
+import scala.util.Random
+
+object IntroFormDemo extends AutoDemo with CssView {
+
+  case class IntroFormDemoModel(minimum: Int, between: Int, maximum: Int)
+
+  object IntroFormDemoModel extends HasModelPropertyCreator[IntroFormDemoModel]
+
+  private val (rendered, source) = {
+    import java.{lang => jl}
+
+    /** The form's model structure.
+      * case class IntroFormDemoModel(minimum: Int, between: Int, maximum: Int)
```

Comment:
This class and its companion could be simply defined here (instead of top-level).